CSS change in the nav-bar

codlocker opened the pull request:
Added a design  to the <a> tags of .nav-links class. This is just to make the UI look more awesome :D .

codlocker committed:
CSS change in the nav-bar 

Added a design  to the <a> tags of .nav-links class. This is just to make the UI look more awesome :D .

ingorichter commented:
Thanks for the contribution. I can see the animation underneath the links in the navbar in Safari, but not in Chrome. I'm using Chrome 54 on OSX. Do you have a chance to check this on Chrome 54?

BTW: did you sign the contributor license agreement (CLA)?

codlocker commented:
I am using Chrome 54 on Windows 7. The animation is working here. 
![image](https://cloud.githubusercontent.com/assets/12949012/19510930/cb468ae2-9602-11e6-8320-3ce8792ba894.png)

codlocker commented:
I am using Chrome 54 on Windows 7. I am getting the animation. I will try it on an OSX machine if I get one.
![chrome54](https://cloud.githubusercontent.com/assets/12949012/19511145/a3ac9e30-9603-11e6-9385-8179a872cbc6.png)

codlocker commented:
Can you give me the link to contributor license agreement (CLA)?

codlocker committed:
Update brackets.io.css

ficristo commented:
I saw you signed the CLA 👍 
Please check better the format of the code: always 4 spaces indentation, braces on the same line, new lines between classes, ...
A personal opinion (you can disregard it) but I would prefer to see the line inside the navbar and not outside.

ingorichter commented:
@codlocker don't look any further. I loaded the page in an incognito session and it worked fine. One of the numerous chrome plugins seems to prevent that animation.

LGTM now.

ingorichter commented:
Please fix the formatting issues as @ficristo mentioned earlier. :-)

codlocker committed:
CSS change in the nav-bar

Added the required formatting to the CSS code and moved animation to within the nav-bar

codlocker commented:
I have fixed the formatting issues and moved the animation to inside the nav-bar as suggested by @ficristo.

ingorichter commented on css/brackets.io.css:
```diff
@@ -316,6 +316,36 @@ blockquote p {
+/* CSS For nav-bar bottom border*/
+@media screen and (min-width:890px)
+{
+    
```
nit: can we remove that blank line?

ingorichter commented:
LGTM. Waiting for the removal of the blank line and then I'll merge it.

codlocker committed:
Update brackets.io.css

codlocker commented:
Done @ingorichter

codlocker committed:
Removed the blank line

I was unable to understand earlier which blank line to remove. I guess this is the one you wanted.

codlocker commented:
Done @ingorichter, @ficristo

ingorichter commented on css/brackets.io.css:
```diff
@@ -316,9 +316,8 @@ blockquote p {
-@media screen and (min-width:890px)
-{    
+@media screen and (min-width:890px) {/* CSS For nav-bar border animation*/
+    
```
I thought this was gone ;-)

codlocker replied:
@ingorichter Do I have to do any more changes?

ficristo replied:
yep, remove that empty line.

codlocker committed:
Removed the empty line

ficristo commented:
The last nits and then I can merge (I promise).
Could you use the this style: `prop: value; /* comment */`
No space before the colon, one after, one after the semicolon, one space before and after the comment.
Thank you for your patience.

codlocker committed:
The last nits

ficristo commented:
Thank you.
